test/ngSanitize/sanitizeSpec.js

```diff
@@ -237,11 +237,9 @@ describe('HTML', function() {
       .toEqual('');
   });
 
-  if (isChrome) {
-    it('should prevent mXSS attacks', function() {
-      expectHTML('<a href="&#x3000;javascript:alert(1)">CLICKME</a>').toBe('<a>CLICKME</a>');
-    });
-  }
+  it('should prevent mXSS attacks', function() {
```

Comment:
TBH, I don't think this test does what it says it does, but that is not related to this PR (it was like that before).
(It is more like `it should not be too strict in preventing mXSS aattacks` :smiley:)